### DISCUSSION TITLE
MCUBoot code with Zephyr for NXP i.MX RT 116x EVK

### DISCUSSION THREAD

**Discussion by vishnusudhan** [zephyrproject-rtos/zephyr#45191] [General]
Hi all,

I am new to this platform, Is there any steps to download the MCUboot with zephyr RTOS on windows for NXP RT116x EVK? please suggest..

**Comment by DerekSnell** [comment]
Hi @vishnusudhan ,
I have not tried MCUboot on the RT1160 platform.  But I have used MCUboot with the mcumgr tool on the similar board MIMXRT1060-EVK, and I shared those steps for you at [MCUboot and Zephyr on NXP i.MX RT series #45252](https://github.com/zephyrproject-rtos/zephyr/discussions/45252).

Let us know how that works for you on the RT1160.  Best regards
